pkg/storage/unified/apistore/prepare.go

```diff
@@ -89,5 +90,27 @@ func (s *Storage) prepareObjectForUpdate(ctx context.Context, updateObject runti
 	if err != nil {
 		return nil, err
 	}
+	return s.handleLargeResources(ctx, obj, buf)
+}
+
+func (s *Storage) handleLargeResources(ctx context.Context, obj utils.GrafanaMetaAccessor, buf bytes.Buffer) ([]byte, error) {
+	if buf.Len() > 1000 {
```

Comment:
@JohnnyQQQQ -- very dumb now... it writes everything (even when blob store is not implemented!)

Reply:
FYI, this is where I think we can potentially register a function that will manage this conversion with knowledge of the the schema and real types

Reply:
@JohnnyQQQQ -- how do you feel about merging this with  `if false {` and then exploring better options for https://github.com/grafana/grafana/pull/94533 in a new PR?